Automattic/WP-Job-Manager
# Add upsell for Job Tags in block editor

Fixes https://github.com/Automattic/WP-Job-Manager/issues/2584

### Changes Proposed in this Pull Request

* Adds a Job Tags upsell panel

### Testing Instructions

* Apply this patch
* Build the assets
* Go to a job listing
* Toggle the Job Tags plugin to make sure the upsell is gone when the plugin is active.

### Screenshot / Video

![lZgnW0.png](https://github.com/Automattic/WP-Job-Manager/assets/3220162/a98707e8-cc2c-404f-8c98-ffb9805be905)






----

| Plugin build for 2fed748aeb2c1ca4661d402c4e1a3a2c352851cc <a href="#"><img width=600></a> |
| ------------------------------------------------------------ |
| 📦 [Download plugin zip](https://wpjobmanager.com/wp-content/uploads/2023/10/wp-job-manager-zip-2618-2fed748a.zip)                       |
| ▶️ [Open in playground](https://wpjobmanager.com/playground/?core=2023/10/2618-2fed748a)             |
